src/_guides/language/language-tour.md

```diff
@@ -403,14 +402,13 @@ You can't change the value of a `const` variable:
 baz = [42]; // Error: Constant variables can't be assigned a value.
 ```
 
-As of Dart 2.5, you can define constants that use
+You can define constants that use
 [type checks and casts](#type-test-operators) (`is` and `as`),
 [collection if](#collection-operators),
 and [spread operators](#spread-operator) (`...` and `...?`):
 
 <?code-excerpt "misc/lib/language_tour/variables.dart (const-dart-25)"?>
 ```dart
-// Valid compile-time constants as of Dart 2.5.
```

Comment:
To fix the build we'll need to update the source of this sample, too. (examples/misc/lib/...)

Reply:
Great catch. Fixed in subsequent commit.